### DISCUSSION TITLE
version 1.11.2 is no longer compatible with phpunit 10

### DISCUSSION THREAD

**Discussion by julienlegrand07** [doctrine/DoctrineBundle#1788] [Q&A] [answered]
Good morning,

I was wondering why remove phpunit 10 compatibility on version 2.11.2?

It's just to understand

https://github.com/doctrine/DoctrineBundle/commit/ecfd2b3f2010fa28c9d380b703575de14c98691d#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34L53

Have a good day

**Comment by stof** [comment, marked as answer]
This commit changes the *dev* dependencies only. It is not about compatibility of the bundle.

We changed that because the symfony phpunit bridge does not actually support PHPUnit 10 so our _testsuite_ cannot be run with PHPUnit 10 while ensuring the same checks than with PHPUnit 9.6.

**Reply by ostrolucky** [reply]
I'll just add that phpunit 10 wasn't supported before either. Despite our constraint allowing phpunit 10, constraints of our dependencies didn't allow it. What happened in that commit was that since we updated those dependencies, phpunit 10 started to get installed and it was causing our tests to break.
